frontend/src/components/Header.tsx

```diff
@@ -24,6 +25,20 @@ type Props = {
   locale?: string;
 };
 
+const NAV_PATHS = ["/search", "/process", "/research", "/subscribe"];
+const homeRegexp = /^\/(?:e[ns])?$/;
+
+const getCurrentNavItemIndex = (currentPath: string): number => {
+  if (currentPath.match(homeRegexp)) {
+    return 0;
+  }
+  const index = NAV_PATHS.findIndex((navPath) => {
+    return currentPath.match(new RegExp(`^(?:/e[ns])?${navPath}`));
```

Comment:
Awesome, that this supports "/path/*" so that nested paths keep the same active header!